core/src/main/res/values/arrays.xml

```diff
@@ -277,4 +277,26 @@
         <item>@string/select_all_above</item>
         <item>@string/select_all_below</item>
     </string-array>
+
+    <string-array name="back_button_behavior_options">
+        <item>@string/back_button_default</item>
+        <item>@string/back_button_go_to_page</item>
+        <item>@string/back_button_open_drawer</item>
+        <item>@string/back_button_double_tap</item>
+        <item>@string/back_button_show_prompt</item>
+    </string-array>
+
+    <string-array name="back_button_behavior_values">
+        <item>default</item>
+        <item>page</item>
+        <item>drawer</item>
+        <item>doubletap</item>
+        <item>prompt</item>
+    </string-array>
+
+    <string-array name="back_button_go_to_pages">
+        <item>@string/queue_label</item>
+        <item>@string/episodes_label</item>
+        <item>@string/subscriptions_label</item>
```

Comment:
We already have an array (not resources) somewhere that allows getting all sidebar fragments. This allows to add the fragments in one place without having to change this file. I think there is also a method that allows to get the title of those fragments.

Not sure where the array can be found, but I think it's somewhere around the sidebar setup.

Reply:
I originally decided to use a new array instead of using the existing `nav_drawer_titles` array because I don't see a use case for going back to fragments like Playback history and Add podcast by default.